.github/workflows/cts_ci.yml

```diff
@@ -56,7 +56,7 @@ jobs:
       matrix:
         include:
           - sycl-impl: dpcpp
-            version: 1dbee22f9c8a3a825deb871bab76937e04fa26fc
+            version: 33e5b101fadc4259135231f99ab62d077b284810
```

Comment:
Is this related to the issue?

Reply:
@keryell, DPC++ version uplift is required to pass updated test. The version we use today doesn't pass as you can see in pre-commit results.

@lbushi25, DPC++ version uplift can be validated only for the local branches. I'll open a new PR using a local branch to validate the uplift status.

Reply:
https://github.com/KhronosGroup/SYCL-CTS/pull/880 should carry the required changes to intel/llvm DPCPP.